src/yargs-static.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+module.exports = function commonArgs(yargs) {
+  return yargs
+    .option('static', {
+      describe: 'How to create the static action.',
+      type: 'string',
+      choices: ['both', 'bind', 'build'],
```

Comment:
- what's the benefit of `both` ?

- why not using an array?
```
hlx deploy --static=bind --static=build
```

Reply:
I expect the `deploy` part of the static refactoring to land before the `publish` part. With `both`, you can switch more easily, because you will have both options deployed way before you start using the new publish feature.

Reply:
?? the linking should also be done during the `deploy` phase. to be consistent with the rest of the actions.

Reply:
When you use `both`, you will have two resources available:

1. `/helix--static`
2. `/helix-services/static@v1`

which one to use depends on the VCL `helix-publish` creates. So the actual choice is happening in the `publish` step.

Reply:
> the *linking* should also be done during the `deploy` phase. to be consistent with the rest of the actions.

*binding*, not linking, yes.

Reply:
> When you use `both`, you will have two resources available:

I thought we should get rid of the private statics....

> which one to use depends on the VCL `helix-publish` creates. So the actual choice is happening in the `publish` step.

if we would put all the actions (even the private ones, if we decide to keep it) below `$actionpackage/helix--static`, the VLC could use the same logic for both cases.

> _binding_, not linking, yes.

I think both should be done during `deploy`:
- _binding_ (ensure that the helix-shared package is available in the customers namespace)
- _link_ (create the sequence action at `$actionpackage/helix--static`, analogous to create the normal script actions.